modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/java/com/liferay/headless/commerce/delivery/catalog/internal/dto/v1_0/converter/SkuDTOConverter.java

```diff
@@ -153,6 +159,50 @@ public Sku toDTO(DTOConverterContext dtoConverterContext) throws Exception {
 						return commercePriceConfiguration.
 							displayDiscountLevels();
 					});
+				setReplacementSku(
+					() -> {
+						if (replacementCPInstance == null) {
+							return null;
+						}
+
+						CPDefinition replacementCPDefinition =
+							replacementCPInstance.getCPDefinition();
+
+						return new ReplacementSku() {
+							{
+								sku = replacementCPInstance.getSku();
+								setProductConfiguration(
+									() -> {
+										if (replacementCPDefinition == null) {
+											return null;
+										}
+
+										return _productConfigurationDTOConverter.
+											toDTO(
+												new DefaultDTOConverterContext(
+													_dtoConverterRegistry,
+													replacementCPDefinition.
+														getCPDefinitionId(),
+													cpSkuDTOConverterConvertContext.
+														getLocale(),
+													null, null));
+									});
+								setSkuId(
```

Comment:
Why do we need the setter for this property? Can't we just set it like the `sku` property?